WireAuthentication/Sources/WireAuthenticationUI/Views/Login/LoginViaEmailOnPrem/LoginViaEmailOnPremView.swift

```diff
@@ -0,0 +1,234 @@
+                    emailField
+                    passwordField
+                    submitButton
+                    forgotPasswordButton
+                    if viewModel.canCreateAccount {
+                        createAccount
+                    }
+                }
+            }
+            .navigationTitle(L10n.CloudUserLogin.title)
+            .navigationBarTitleDisplayMode(.inline)
+            .padding(32)
+            .background(ColorTheme.Backgrounds.surface.color)
+            .cornerRadius(16)
+            .overlay(
+                RoundedRectangle(cornerRadius: 16)
+                    .stroke(ColorTheme.Backgrounds.surface.color, lineWidth: 1)
+            )
+        }
+        .presentationDetents(viewModel.hasProxySupport ? [.large] : [.medium, .large])
+        .interactiveDismissDisabled()
+        .presentationDragIndicator(.hidden)
+        .onChange(of: password) { newPassword in
+            showPasswordRules = !viewModel.isValidPassword(newPassword)
+        }
+    }
+
+    @ViewBuilder private var welcomeMessage: some View {
+        VStack(spacing: 14) {
+            Button(action: {
+                showCustomBackendAlert.toggle()
+            }, label: {
+                Text(L10n.OnPremUserLogin.title(viewModel.backendName) + " ")
+                    .foregroundColor(ColorTheme.Buttons.Secondary.onEnabled.color)
+                + Text(Image(systemName: "info.circle"))
+                    .foregroundColor(.gray)
+            })
+            .multilineTextAlignment(.center)
+            .font(.textStyle(.h2))
+            .lineLimit(nil)
+            .fixedSize(horizontal: false, vertical: true)
+            .alert(L10n.OnPremUserLogin.Alert.title, isPresented: $showCustomBackendAlert) {
+                Button(L10n.OnPremUserLogin.Alert.button, role: .cancel) {}
+            } message: {
+                Text(viewModel.backendInfo)
+            }
+            Text(L10n.OnPremUserLogin.message)
+                .multilineTextAlignment(.center)
+                .wireTextStyle(.body1)
+                .lineLimit(nil)
+                .fixedSize(horizontal: false, vertical: true)
+        }
+    }
+
+    @ViewBuilder private var emailField: some View {
+        LabeledTextField(
+            placeholder: nil,
+            title: L10n.CloudUserLogin.InputEmail.title,
+            string: .constant(viewModel.email)
+        )
+        .disabled(!viewModel.email.isEmpty)
+    }
+
+    @ViewBuilder private var passwordField: some View {
+        PasswordField(
+            password: $password,
+            placeholder: L10n.CloudUserLogin.InputPassword.placeholder,
+            title: L10n.CloudUserLogin.InputPassword.title,
+            passwordRules: viewModel.localizedPasswordRules,
+            isValidPassword: viewModel.isValidPassword
+        )
+    }
+
+    @ViewBuilder private var submitButton: some View {
+        Button(action: {
+            viewModel.submitPassword(password)
```

Comment:
I would expect that we submit not only the account password but also the proxy credentials it there are any.

Reply:
I created https://wearezeta.atlassian.net/browse/WPB-16256 to implement Proxy support